CONTRIBUTING.md

```diff
@@ -22,6 +22,17 @@ systems as possible.
 
 Changes to the project may only be merged if the CI finishes successfully.
 
+## Releases
+
+New releases are created by
+
+* adding a commit that
+  * updates the project version in [meson.build](meson.build)
+  * updates the [changelog](CHANGELOG.md) to the current version while adding a
+    new (empty) subsection for `[Unreleased]` modifications with `Added`,
+    `Changed` and `Deleted` subsubsections.
+* tagging the commit using a signed tag according to the new project version
+
```

Comment:
syntax and version scheme should be mentioned here.
do we prefix the tag with "v" ? do we use semantic versioning, year-month, something else? no odd number releases mean unstable/testing?

Reply:
I would probably just stick with semver, but it does not have much of a meaning in this situation of course.
Maybe just use full numbering versions (e.g. 1, 2, 3) instead of 0.1.0?
I guess calver is also fine, but I believe a major version focused versioning might be fine as well (after all calver is more or less that anyways).

In regards to the `v` prefix, I don't believe it is necessary at all, but I think github promotes it as "the thing to do" for some reason :shrug:

Reply:
We will settle for simple major version increases (e.g. `1`, `2`, `3`).